keras/requirements.txt

```diff
@@ -1,7 +1,4 @@
-# TODO(hvy): Remove the version constraint. See https://github.com/optuna/optuna/pull/2851
-keras<2.6.0
+keras
```

Comment:
note: we can remove this line since keras is installed via the installation of Tensorflow.